### PR TITLE
Add service command

### PR DESCRIPTION
**Pull request by sitaramshelke**
I have added command to add a new service with pretty basic test code. Let me know if we should write more tests.

### CODE REVIEW

File: commands/add_service.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+	"github.com/cristianoliveira/ergo/proxy"
+)
+
+// AddService Allows to add new services to existing configuration file
+// Usage:
+// `ergo add service servicehost:port`
+func AddService(config *proxy.Config, service proxy.Service, configFile string) {
+	old_service := config.GetService(service.Name + config.Domain)
+	if old_service != nil {
```
Comment (cristianoliveira) [inline commands/add_service.go]: Good catch! I was thinking if you had validated this 👍

File: commands/add_service.go [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	"fmt"
+
+	"github.com/cristianoliveira/ergo/proxy"
+)
+
+// AddService Allows to add new services to existing configuration file
+// Usage:
+// `ergo add service servicehost:port`
+func AddService(config *proxy.Config, service proxy.Service, configFile string) {
+	old_service := config.GetService(service.Name + config.Domain)
```
Comment (srini156) [inline commands/add_service.go]: @sitaramshelke - rename `old_service` to `oldService` for `golint`. Can you additionally run `golint` for `add_service.go`?

Comment (sitaramshelke) [reply commands/add_service.go]: @srini156 Good find. I actually I had setup VSCode to use golint. Will check the settings as well.
Thanks.

**Comment by sitaramshelke** [comment]
@cristianoliveira I am thinking once this PR looks good to you, we can squash the commits into single commit. What do you think?

**Comment by cristianoliveira** [comment]
Hey @sitaramshelke Sorry for the delayed answer! Today was an uncommon busy day here. Tomorrow is a holiday so I can answer faster! Regarding the squash, I can do it here.

**Comment by cristianoliveira** [comment]
Thanks again for your contributions!

**Pull request merged**